src/errors/errors/cli.ts

```diff
@@ -22,12 +25,12 @@ export function addOclifExitCode(error: Record<string, any>, options?: {exit?: f
 }
 
 export class CLIError extends Error implements OclifError {
-  code?: string
+  code?: string | undefined
```

Comment:
what's the value of having both `?` and the `| undefined` ?

Reply:
this https://github.com/oclif/core/issues/960